generators/generator-base-private.js

```diff
@@ -786,22 +786,6 @@ module.exports = class extends Generator {
         });
     }
 
-    /**
-     * Check if git connection can be established
-     */
-    checkGitConnection() {
```

Comment:
Though, we might not use this directly. But this ensures that the user is having `git` and we can execute `git` based comments. I think it is better to have this check and not to relay on the internet to check

Reply:
the function checkGit already exists and is just above :p

Reply:
😬